common-utils/src/common/deployment/__init__.py

```diff
@@ -131,10 +117,7 @@ def get_pyproject_metadata() -> PyProjectMetadata:
     )
 
 
-PYPROJECT_METADATA = get_pyproject_metadata()
-
-
-def create_image_name_str(env_name: str, python_version: str) -> str:
+def create_image_name_str(project_name: str, env_name: str, python_version: str) -> str:
```

Comment:
removing the pyproject.toml get means we need to rely on the function caller to provide project_name